### ISSUE TITLE
404 Error on catalog download

### ISSUE THREAD

**Issue by reckoner** [JnyJny/springer_downloader#5]
This URL 

https://resource-cms.springernature.com/springer-cms/rest/v1/content/17858272/data/v4

returns 404 error.

**Comment by JnyJny** [comment]
Yup, I confirm the 404. It looks like the URL version has been rev'ed. As soon as I confirm that incrementing v4 to v5 fixes the problem I'll push a fix.

**Comment by JnyJny** [comment]
Yup.

```console
springer refresh-catalog -u https://resource-cms.springernature.com/springer-cms/rest/v1/content/17858272/data/v5
📚|en-all
```

I'll update the URL in the package and publish a new revision to PyPI.

**Comment by JnyJny** [comment]
Fixed with 35493c378cd904363fe631b6ae15305d7f5b5da9, update to 0.14.1 to get the fix.
